efficientdet/dataset_coverter.py

```diff
@@ -0,0 +1,55 @@
+# function converts dataset_to_convert to match categories in dataset_template
```

Comment:
Please change the structure of this file to improve readibility and prevent unnecesary code execution when code is imported.
Proposition of structure:
- imports
- functions
- `if __name__ == "__main__":` clausule
     - parameters eg. epi_path and taco_path
     - execution of the script (loading files, writing files)